public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+  
+  The Hero Job Board application uses output event binding to sign that a hero undertakes the announced job. The `HeroPanel` component declares 
+  an output property as a type of `EventEmitter`, through which &mdash; as the type name suggests &mdash; it can emit an event. The parent 
+  `HeroJobBoard` component can use the output property to attach an event handler to that particular event.
+  
+  `HeroPanel` declares the `onJobTaken` output event property, as the `@Output` decorator indicates:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-panel.ts', 'component', 'take-job-event/hero-panel.ts (extract)', {pnk: /(@Output\(\)|takeJob\(\)|onJobTaken\.next)/g})
+
+:marked
+  The `takeJob()` method invokes the `next()` method of `onJobTaken`, and passes the `hero` assigned to this `HeroPanel`. 
+  The parameter of `next()` is the argument of the event emitted. Thanks to the strong type notation in TypeScript, `EventEmitter` is a generic type,
+  so we could declare `onJobTaken` as an `EventEmitter<Hero>`. When we invoke `next`, the TypeScript compiler checks whether the event argument 
+  is compatible with `Hero`.
+  
+  To use `@Output()` and `EventEmitter`, `HeroPanel` needs to import them:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-panel.ts', 'import', null, {blk: /(Output|EventEmitter)/g})
+
+:marked
+  Now our `HeroPanel` is ready to emit its event. `HeroJobBoard` attaches its `heroTakesJob()` event handler method to the `onJobTaken` output
+  event of `HeroPanel` using the output property binding syntax &mdash; wrapping the snake-case name of the output property, `on-job-taken`, into parentheses.
+  The output property binding expression passes the `$event` identifier &mdash; that represents the argument of the emitted event &mdash; to the handler method:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-job-board.ts', 'component', 'take-job-event/hero-job-board.ts', {pnk: /(\(on-job-taken\))/g, blk: /(heroTakesJob\(hero: Hero\))/g})
+
+.alert.is-critical
+  :marked
+    Although this technique works in many situations, it's an **antipattern**, and Angular discourages using it.
+  
+:marked
+  The event argument is the `hero` property value of `HeroPanel` that is passed to the `next()` method of the `EventEmitter` instance beyond
+  `onJobTaken`. `HeroJobBoard` receives the `Hero` instance that undertakes the job, and adds it to its `respondingHeroes` array.
+  
+  With output event binding, we established this communication structure from `HeroPanel` children to `HeroJobBoard`:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/child-to-parent.png" alt="Output event binding")
+  
++makeExample('component-communication/ts/fragments/hero-job-board-with-parentref.ts', null, 'take-job-parentref/hero-job-board.ts (extract)', {pnk: /(\[parent\]|jobBoard\(\))/g})
```

Comment:
This discussion about `parent` binding actually ***belongs under the first technique ... Property Binding***.

It is a way to have bi-directional "communication" with property binding alone.

I don't mind mentioning this approach. We shouldn't go on about it and I do NOT think we should have a sample that does it.  That only encourages the idea.